pkg/cidata/cidata.go

```diff
@@ -369,6 +370,49 @@ func GenerateISO9660(instDir, name string, y *limayaml.LimaYAML, udpDNSLocalPort
 	return iso9660util.Write(filepath.Join(instDir, filenames.CIDataISO), "cidata", layout)
 }
 
+// getHostTimezone tries to get host timezone from env or specific files
+func getHostTimezone() string {
+	// 1. try to get timezone from env.TZ
+	tzEnv := os.Getenv("TZ")
+	if tzEnv != "" &&
+		// skip if TZ is in "std offset dst offset, rule" format
+		!strings.Contains(tzEnv, " ") {
+		return tzEnv
+	}
+	logrus.Debugf("cannot get timezone from env TZ: [%s]", tzEnv)
+
+	// 2. try to get timezone from /etc/timezone
+	tzName, err := os.ReadFile("/etc/timezone")
+	if err == nil {
+		return string(tzName)
+	}
+	logrus.Debugf("cannot get timezone from /etc/timezone: %s", err)
+
+	// 3. try to get timezone from /etc/localtime soft link
+	// base on the assumption that <zoneinfo_basedir>/zone.tab file exists
```

Comment:
Haven't found any official documents about zoneinfo data file hierarchy, just _assume_ `zone.tab` file exists.

Reply:
I don't know either. Right now it seems like a safe assumption, but there are 2 comments in the file that might want us to look for safer alternatives:

```
# tzdb timezone descriptions (deprecated version)
[...]
# This file is intended as a backward-compatibility aid for older programs.
```

Maybe we should look for a directory named `Antarctica` (or a file named `Etc/UTC`) instead?